google-cloud-spanner/src/main/java/com/google/cloud/spanner/Spanner.java

```diff
@@ -36,9 +42,34 @@ public interface Spanner extends Service<SpannerOptions>, AutoCloseable {
    * }</pre>
    * <!--SNIPPET get_dbadmin_client-->
    */
+  @Deprecated
```

Comment:
* Sure, our internal documents mention "Soon to be deprecated" since internally readers may confuse this with the "Deprecation Policy" which means that the code will be actually deleted. To avoid that confusion we mentioned "soon to be deprecated".
* In the Java code, shouldn't we provide a way for our customers to know that these methods will indeed lose support soon. While the code is not deleted, we are providing customers a fair notice that the support for these methods could be removed in future. Unfortunately Java does not provide me a "Soon to be deprecated" annotation, else I agree it would have been a better choice.

I'm ok with removing this annotation, but wanted to know your thoughts on this.

Reply:
Removed the `@Deprecated` annotation from this PR. Will take up the discussion offline and in a new PR.

Reply:
I agree that there is a missing annotation in Java here, that would indicate something like 'you should not use this method for new code anymore, but you are also not in a rush to remove this in all your current code'. Marking this as `@Deprecated` in my mind goes too far, as it is a method that has been around since the very first version, and that is likely to be spread across a very large range of customer code. Marking something as `@Deprecated` is normally a hint that something will be removed soonish, and that users should actively port their code. I expect the latter to cause a lot of questions if we mark it as such without some gentle pre-warning first.